source/server/hot_restarting_base.cc

```diff
@@ -93,11 +104,11 @@ void HotRestartingBase::sendHotRestartMessage(sockaddr_un& address,
     message.msg_iovlen = 1;
 
     // Control data stuff, only relevant for the fd passing done with PassListenSocketReply.
-    uint8_t control_buffer[CMSG_SPACE(sizeof(int))];
+    absl::FixedArray<uint8_t> control_buffer(CMSG_SPACE(sizeof(int)));
```

Comment:
Why these changes? I think you can use std::array in this case also if you need to change it for some reason?

Reply:
Oh forgot to say, clang on OS X didn't like int foo[]... shall I drop it since we don't care about OS X for hot restart?